src/components/inputs/tooltip/tooltip.component.tsx

```diff
@@ -12,7 +12,7 @@ interface TooltipProps {
 const Tooltip: React.FC<TooltipProps> = ({ field }) => {
   const { t } = useTranslation();
   return (
-    <CarbonTooltip align="top" label={t(field.questionInfo)} description={t(field.questionInfo)}>
+    <CarbonTooltip align="right" label={t(field.questionInfo)} description={t(field.questionInfo)}>
```

Comment:
@vasharma05 have u tested this with longer question labels? The tooltip might just overflow to the right side instead

Reply:
Yeah, I was thinking that too, but I couldn't find out the reason the tooltip is getting cropped. I'm looking into it right now

Reply:
@vasharma05, any new developments on this?